### PR TITLE
Create 2023-05-29-Qubit.md

### PR DESCRIPTION
**Pull request by L0nest**
A new article has been added to the "post" directory. The article is about hacking Qubit Finance in 2022.

### CODE REVIEW

**Review by eric-oconnor — APPROVED** [review]
Well-rounded article with easily verifiable facts from multiple sources.

File: content/attacks/posts/2023-05-29-Qubit.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-05-29-Qubit.md:1]: add all required headers

File: content/attacks/posts/2023-05-29-Qubit.md [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+On January 28, 2022, Qubit Finance, a project built on the BNB Chain (formerly known as Binance Smart Chain), announced a breach of its QBridge credit protocol on their Twitter page. The platform was hacked due to a specific vulnerability in the bridge. Hackers were able to mint an unlimited amount of xETH as collateral for loans on the Binance Smart Chain network.
+
+## Attackers
+
+The perpetrators remain unidentified.
+
+## Losses
+
+A huge amount of xETH was issued, used as collateral, and took about $80 million worth of tokens from the pool.
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-05-29-Qubit.md:16]: use specific numbers, avoid words like "huge"

File: content/attacks/posts/2023-05-29-Qubit.md [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+
+## Timeline
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-05-29-Qubit.md:18]: timeline needs to be a timeline, not one event

File: content/attacks/posts/2023-05-29-Qubit.md [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+
+- **January 28, 2022:** PeckShield, a blockchain security and smart contract audit company, confirmed the exploit.
+
+## Security Failure Causes
+
+CertiK Security reported that the attacker called the deposit function in the QBridge contract, but did not deposit funds. Ethereum QBridge recorded the deposit event and released $qXETH to the hacker on the BSC blockchain. The hacker repeated the procedure several times, increasing production to nearly $80 million.
+> "QBridge treats the deposit event as a #ETH replenishment because the 'deposit' and 'depositETH' methods in the QBridge contract fire the same event."
+
+## Lessons Learned From The Attack
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-05-29-Qubit.md:27]: don't create any non-standard section not found in other articles

File: content/attacks/posts/2023-05-29-Qubit.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+
+The Qubit hack was made possible by the fact that a legacy function (deposit) remained in the contract after it was replaced by a newer function (depositETH).  Their overlapping functionality and vulnerabilities that allowed fake deposits made this hack possible.
+
+The existence of legacy code in the contract indicates issues with code review and change management.  Similarly, the vulnerabilities that allowed the fake deposit might have been detected during a smart contract audit.
+
+## Additional Details
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-05-29-Qubit.md:33]: don't create any non-standard section not found in other articles

File: content/attacks/posts/2023-05-29-Qubit.md [hunk 6]
```diff
@@ -0,0 +1,37 @@
+---
+date: 2022-03-23
+title: Hackers Breach Qubit Finance Platform, Stealing $80 Million
+---
+
+## Summary
+
+On January 28, 2022, Qubit Finance, a project built on the BNB Chain (formerly known as Binance Smart Chain), announced a breach of its QBridge credit protocol on their Twitter page. The platform was hacked due to a specific vulnerability in the bridge. Hackers were able to mint an unlimited amount of xETH as collateral for loans on the Binance Smart Chain network.
+
+## Attackers
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-05-29-Qubit.md:10]: add theories, description of their sophistication, anything that can help to narrow the suspect pool

**Review by evgenydmitriev — CHANGES_REQUESTED** [review]

**Comment by evgenydmitriev** [comment]
/qualitycheck

(only wiki reviewers can trigger the quality check)

File: content/attacks/posts/2023-05-29-Qubit.md [hunk 7]
```diff
@@ -0,0 +1,34 @@
+---
+date: 2022-03-23
+custodians: Xplosive Ethereum (xETH)
+categories: Crypto currency attacks, Security breaches
+title: Hackers Breach Qubit Finance Platform, Stealing $80 Million
```
Comment (evgenydmitriev) [inline content/attacks/posts/2023-05-29-Qubit.md]: Quotes are needed since there are special characters in the title

**Review by evgenydmitriev — APPROVED** [review]

**Comment by danisztls** [comment]
@jhirschkorn can you comment `/payout` here to test the automation? AFAIK @albina-at-inca already paid the author.

**Pull request merged**